Ability to detect SQL queries without bind parameters #9 (#32)
quick-perf/quickperf-live: doc/database_diagnostics.md

GET 200 http://localhost:8080/json-with-n-plus-one-select
* [WARNING] You have reached your SQL executions threshold : 11 > 10
```

### Missing binded parameters queries warning

When passing parameters to queries, the generated query should not have the values but a generic binded parameter. This will allow the database to avoid parsing and executing the same query over and over again. QuickPerf-live can help you detect unbound parameters queries. All you need to do is to activate the following property  :

:wrench: _.properties_ configuration example

```
quickperf.database.sql.without-bind-param.detected=true
```

:wrench: YAML configuration example

```yaml
quickperf:
  database:
    sql:
      without-bind-param: true
```

:wrench: MBean configuration

```
QuickPerf
  -- Database
      -- Operations
           -- boolean isSqlWithoutBindParamDetected()
           -- void setSqlWithoutBindParamDetected(boolean)
```

:mag_right: Log example

```
Hibernate: SELECT * FROM Player p where p.lastName like '%ronaldo%'
2022-12-14 13:59:19.776  WARN 13632 --- [qtp640585435-24] s.QuickPerfHttpCallHttpCallWarningLogger : 
GET 200 http://localhost:8080/player/unbounded-params?name=ronaldo

> 	* [WARNING] Some of your SQL queries does not use bound parameters.

Reviewer: ```suggestion
	* [WARNING] Some of your SQL queries don't  use bind parameters.
```
